automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/methodrepo/MethodRepoSpec.scala

```diff
@@ -36,11 +36,32 @@ class MethodRepoSpec extends FreeSpec with WebBrowserSpec with Matchers {
 
       methodRepoPage.ui.hasMethod(namespace, name) shouldBe true
 
-      // go back into it and redact
+      api.methods.redact(namespace, name, 1)
```

Comment:
If this is clean-up, it should be immediately after `createDialog.createMethod`. Otherwise any test error or assertion failure will cause it to be skipped. It can be deferred until the test is complete with `register cleanUp`:

```
register cleanUp api.methods.redact(namespace, name, 1)
```